contributing.md

```diff
@@ -98,16 +101,13 @@ We use the [semantic versioning scheme](https://semver.org/). The semver implies
 
 When incrementing the version number, the following tasks are required:
```

Comment:
Are we *actually* using semver schema? I thought at some point in our discussions it was said that we weren't using it?

Reply:
@awav ?